group_test.go

```diff
@@ -234,21 +186,15 @@ func TestGroup(t *testing.T) {
 		{
 			name: "Resource Index",
 			setup: func(req *http.Request) {
-				mockConfig.On("Get", "cors.paths").Return([]string{}).Once()
-				mockConfig.On("GetString", "http.tls.host").Return("").Once()
-				mockConfig.On("GetString", "http.tls.port").Return("").Once()
-				mockConfig.On("GetString", "http.tls.ssl.cert").Return("").Once()
-				mockConfig.On("GetString", "http.tls.ssl.key").Return("").Once()
```

Comment:
Don't use `GlobalMiddleware` (line 244) directly, so these lines can be removed.